cfme/tests/openstack/infrastructure/test_roles.py

```diff
@@ -21,67 +20,69 @@
          'CephStorage']
 
 
+@pytest.yield_fixture(scope="module")
+def roles(appliance, provider):
+    collection = DeploymentRoleCollection(appliance=appliance)
+    view = navigate_to(collection, 'All')
+    roles = [collection.instantiate(name=item.name, provider=provider)
+             for item in view.entities.get_all()]
```

Comment:
**REQUIRED**
It's better to have this inside collection class as `all()` method

Reply:
ok, the line of code I am adding in the collection

Reply:
@quarckster I added  "all()" method in the collection. It will return all roles available on All Page.